storehaus-mongodb/src/main/scala/com/twitter/storehaus/mongodb/MongoStore.scala

```diff
@@ -93,24 +96,24 @@ class MongoStore[K: MongoValue, V: MongoValue: Manifest] (
 
   protected def getValue(valueObject: MongoDBObject): Option[V] = {
```

Comment:
looks good to me, I'd just add:
```scala
s"Cannot convert $value of type ${value.getClass} to ${manifest[V]}"
```
so we can see the full value and type in the exception message.